src/content/getting-started/secure-your-app/_index.md

```diff
@@ -1,29 +1,159 @@
+
+In this guide, you will learn how to make your application compliant with the secure baseline and how create policy exceptions when needed.
 
 ## Requirements
 
-<requirements>
+Before start, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}).
+
+A part, you need to deploy an application, here `hello-world` application is used, to the workload cluster. For explanation on how to deploy the application please read the [previous step]({{< relref "/getting-started/install-an-application" >}}).
+
+## Step 1: Understand the security baseline
+
+By default the clusters come with a `restricted` set of policies. You can read more about which policies are part of the `restricted` set in the [upstream documentation](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted). The `restricted` set of policies are applied to all the namespaces in the cluster.
+
+As brief summary, the `restricted` set of policies includes:
+
+- Containers only can run with `non-root` users
+- Not allowing privileged containers
+- Not allowing host namespaces
+- Allowing only a set of capabilities (indeed, only `NET_BIND_SERVICE` is allowed)
+- `HostPath` volumes aren't allowed
+- Only default `Seccomp` and `AppArmor` profiles are allowed
+- `ReadOnlyRootFilesystem` is enforced
+
+In case you application needs to use any of the features that aren't allowed by the `restricted` policies, you need to create a policy exception.
+
+## Step 2:  Configure the security context
+
+In the `hello-world` application, the `restricted` policies aren't violated. The application is running with a `non-root` user and it's not using any special capability of the system.
+
+In `Kubernetes`, the security context is defined in the `Pod` resource. Even you don't need a special security configuration, you still need to a minimal security context to your `Pod` resource. Inside the `Pod` resource, you find a general `securityContext` which are applied to all the containers within the resource. A part, you can define a `securityContext` for each container.
+
+Inspecting the `hello-world` application, you can see pods are created without general `securityContext` but with a `securityContext` for the container.
+
+```yaml
+spec:
+  containers:
+  - name: hello-world
+    ...
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      runAsNonRoot: true
+      runAsUser: 1000
+      seccompProfile:
+        type: RuntimeDefault
+```
+
+Those are necessary to be compliant and pass the validation of the policy engine.
+
+## Step 3:  Create a policy exception
+
+Let's decide the `hello-world` application needs to run as a `root` user because the container image is designed to run as a `root` user you don't have more time and your CTO is pushing you to deploy the application.
```

Comment:
```suggestion
Let’s assume the `hello-world` application needs to run as a `root` user because the container image is designed this way. You don't have more time and your CTO is pushing you to deploy the application.
```